### PR TITLE
Implement LogOut button functionality and redirect from '/' to '/login'

### PR DESCRIPTION
**Pull request by RokoVarano**
- [x] Logout button in sidebar takes the user to '/login'
- [x] '/' path is automatically redirected to '/login'
- [x] using the Logout button clears localstorage
- [x] using the Logout button reverts the user state to an empty object

### CODE REVIEW

File: src/components/Sidebar.js [hunk 1]
```diff
@@ -33,7 +36,7 @@ function Sidebar() {
             </NavLink>
           </div>
           <div className="text-center row mt-auto pb-5">
-            <NavLink to="/" className="nav-link text-dark">
+            <NavLink to="" type="button" onClick={() => dispatch(logOut())} className="nav-link text-dark">
```
Comment (notlfish) [inline src/components/Sidebar.js]: `logOut` is a function that gets the dispatch method as an argument. Please, change this to
```javascript
onClick={() => dispatch(logOut)}
```
that is, without calling logOut. As it is now, it works, but it gives an error in the console.

**Review by notlfish — CHANGES_REQUESTED** [review]
# Changes Requested
### Good work on implementing the log-out functionality!

Your pull request is working as it should, but it throws an error that can easily be fixed. There's more info in the comments.
Please, fix this before merging

File: src/components/Sidebar.js (hunk 1 above)
Comment (RokoVarano) [reply src/components/Sidebar.js]: Change has been implemented. Thanks for the feedback!

**Review by notlfish — APPROVED** [review]
# Approved
Good job on implementing the requested changes, you can now merge this branch.

**Pull request merged**